corehq/apps/app_manager/models.py

```diff
@@ -2592,7 +2592,7 @@ def validate_detail_columns(self, columns):
                 domain = self.get_app().domain
                 project = Domain.get_by_name(domain)
                 try:
-                    if not should_sync_hierarchical_fixture(project):
+                    if not should_sync_hierarchical_fixture(project, self.get_app()):
```

Comment:
is this memoized?

Reply:
Nope: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/locations/fixtures.py#L197

Reply:
Oops. that was unclear. I meant is `self.get_app()` memoized? (I assume it is, but did see the call to it a couple lines above)

Reply:
Ah I see, then yeah as it says a couple lines up, it's not memoized: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/models.py#L2565
If it were memoized, would it mess this up?

Reply:
Nope that looks fine. looks like that property gets set and this won't incur an extra db lookup

Reply:
ok cool